### PR TITLE
Testing proof-of-concept

### PR DESCRIPTION
**Pull request by tgross**
Provides a simple test of scaling up and scaling down, checking that the Nginx configuration is updated correctly. Depends on the work in https://github.com/autopilotpattern/testing/pull/1

cc @misterbisson this is still a work in progress I think

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+build:
+	cd tests && docker build -t="test" .
+
+test-triton:
+	docker run --rm \
+		-e DOCKER_TLS_VERIFY=1 \
+		-e DOCKER_CERT_PATH=/.sdc/docker/$(SDC_ACCOUNT) \
+		-e DOCKER_HOST=$(DOCKER_HOST) \
+		-v $(HOME)/.sdc:/.sdc \
+		-v $(shell pwd):/src \
```
Comment (tgross) [inline Makefile]: This obviously doesn't quite work as-is. We need a way to inject credentials into the running container.

**Comment by misterbisson** [comment]
Looks good enough to merge end fix whatever we discover later.

**Comment by tgross** [comment]
Once https://github.com/autopilotpattern/testing/pull/4 is merged I think this is ready to go in as well.

**Comment by misterbisson** [comment]
🏡 🚶

**Pull request merged**